pkg/minikube/sysinit/systemd.go

```diff
@@ -123,7 +125,14 @@ func (s *Systemd) Stop(svc string) error {
 
 // ForceStop terminates a service with prejudice
 func (s *Systemd) ForceStop(svc string) error {
```

Comment:
consider adding this to openRC too
https://github.com/medyagh/minikube/blob/1bac30bbe7caa5b8c64b35cfb97c197ffe08840a/pkg/minikube/sysinit/openrc.go#L166

Reply:
Not sure what to do there. This PR is targeting a very specific systemd issue